apps/music/elements/music-controls.js

```diff
@@ -113,8 +113,8 @@ proto.createdCallback = function() {
     }
   );
 
-  this.els.container.addEventListener('click', (evt) => {
-    var button = evt.target.closest('button');
+  this.els.container.addEventListener('click', throttle((evt) => {
```

Comment:
I've been thinking about this; do we actually *need* to throttle the next/previous buttons to fix this bug? I can see a user wanting to tap "next" twice to quickly skip two songs ahead.